src/awsexplorer/regionNode.ts

```diff
@@ -17,22 +17,22 @@ import { AWSTreeNodeBase } from '../shared/treeview/nodes/awsTreeNodeBase'
  * an account's Lambda Functions and CloudFormation stacks for this region)
  */
 export class RegionNode extends AWSTreeNodeBase {
-    private info: RegionInfo
+    private region: Region
     private readonly childNodes: AWSTreeNodeBase[] = []
 
     public get regionCode(): string {
```

Comment:
Are regionCode and regionName used elsewhere? If not, I'd dump them and just use region.id and region.name directly (you're already doing that throughout this PR)

Reply:
RegionNode's interface can remain as is for this change. 
`id` is a TreeItem property (part of the class parent chain)

Reply:
Fair, but at the very least be consistent within the class (get rid of the `this.regionName` and `this.regionCode` in the update function). Long term, if we're using `region.id` and `region.name`  elsewhere, we should just have this node return a `Region` (out of scope for this though)

Reply:
RegionCode is used elsewhere. A consistency pass can be made separate from this PR.
Class properties are commonly used as an encapsulation measure. Updating code away from them is not an ideal practice.

This feels like bikeshedding.

Reply:
NVM, I'm misreading; the `region.name` calls elsewhere here aren't tied to the `this.region`.